sw/device/silicon_creator/lib/drivers/kmac_functest.c

```diff
@@ -19,12 +20,32 @@
 
 OTTF_DEFINE_TEST_CONFIG();
 
+enum{
+  SHORT_DIGEST_LEN = 8,
+  LONG_DIGEST_LEN = 75,
+  SHORT_MSG_LEN = 13,
+  LONG_MSG_LEN = 26 * 4,
+};
+
+typedef struct unalignedhash{
+  char unused;
+  char short_payload[SHORT_MSG_LEN];
+  char long_payload[LONG_MSG_LEN];
+} unalignedhash_t;
+
+static alignas(uint32_t) unalignedhash_t kKMAC_test_data = {
```

Comment:
This should be `static const` and the use the same naming constant naming convention: `kKmacTestData`.